core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2677,6 +2677,16 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey SECURITY_LOGIN_IMPERSONATION_USERNAME =
+      new Builder(Name.SECURITY_LOGIN_IMPERSONATION_USERNAME).setDescription(String.format(
+          "When alluxio.security.authentication.type is set to SIMPLE or "
```

Comment:
nit: use `PropertyKey.Name.SECURITY_AUTHENTICATION_TYPE` rather than the raw string